src/features/landing/components/Header/HeaderActionButtons/HeaderActionButtons.tsx

```diff
@@ -0,0 +1,59 @@
+import { Grid } from '@mui/material';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Button from '@/components/ui/Button/Button';
+
+import useScreenSize from '../../../hooks/useScreenSize/useScreenSize';
+import { TRANSLATION_NAMESPACE } from '../../../utils/constants/constants';
+
+interface IHeaderActionButtonsProps {
+  onSignInButtonClick: () => void;
+  onTryItOutButtonClick: () => void;
+}
+
+export default function HeaderActionButtons({
+  onSignInButtonClick,
+  onTryItOutButtonClick,
+}: IHeaderActionButtonsProps) {
+  const { t } = useTranslation(TRANSLATION_NAMESPACE);
+  const { isMobile, isSmallest, isSmallTablet, isBigTablet } = useScreenSize();
+
+  if (isMobile || isSmallest || isSmallTablet || isBigTablet) {
+    return null;
```

Comment:
why must be returned null? difficult to understand for what reason it must return null